irohad/ametsuchi/impl/flat_file/flat_file.hpp

```diff
@@ -1,28 +1,15 @@
 /**
- * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
- * http://soramitsu.co.jp
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef IROHA_FLAT_FILE_HPP
-#define IROHA_FLAT_FILE_HPP
+#ifndef IROHA_FLAT_FILE_IMPL_HPP
```

Comment:
Please rename to `IROHA_FLAT_FILE_HPP`, so that it is consistent with filename. Same in the end of this file.